changelog.md

```diff
@@ -2,6 +2,7 @@
 
 ## v0.6.3 | t.b.d.
 
+* Add `align` kwarg to `CorrelatedStack`. If enabled, this option will automatically align the color channels for multi-channel images, provided that the alignment metadata from Bluelake is available.
```

Comment:
Very minor, but it seems that two things were actually added:
* Add support for RGB images to `CorrelatedStack`.
* Add image alignment to `CorrelatedStack`. Image alignment is enabled by default and will automatically align the color channels for multi-channel images, provided that the alignment metadata from Bluelake is available. Image alignment can be disabled by specifying `align=False` when loading the `CorrelatedStack`.